vnext/src-win/Libraries/Components/Xaml/FabricXamlControl.windows.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @format
+ * @flow
+ */
+
+const nativeComponentRegistry = require('react-native/Libraries/NativeComponent/NativeComponentRegistry');
+
+const StackPanel = nativeComponentRegistry.get('StackPanel', () => ({
```

Comment:
If we had spec files for these, then we'd be able to export them, and we'd get better type information for the users of these components.